app/src/components/assistant-ui/thread-list.tsx

```
@@ -783,120 +727,148 @@ const HierarchicalThreadList: FC<HierarchicalThreadListProps> = ({
                             if (currentThread.status === "archived") {
                                 updateThread(currentThreadId, model, "active");
                             } else {
                                 updateThread(currentThreadId, model, "archived");
                             }
                         }
                     }
-                    break;
 
-                case "ArrowUp":
+                    break;
+                }
```

ESLint

[error] Expected a 'break' statement before 'case'.

(no-fallthrough)